modules/solid_mechanics/test/tests/interaction_integral_benchmark/tests

```diff
@@ -1,9 +1,10 @@
 [Tests]
  [./test]
-   type = 'Exodiff'
+   type = 'CSVDiff'
    input = 'input.i'
-   exodiff = '360degree_model_out.e'
+   csvdiff = '360degree_model_out.csv'
    abs_zero = 1e-7
    max_parallel = 1           # nl_its and lin_its will not be the same in parallel and serial
+   allow_warnings = true
```

Comment:
You shouldn't need to allow this warning either, should you?  If you move 'disp_x' etc. from the GlobalParams block, you shouldn't get the warning about the old syntax.

Reply:
@sveerara This is the only issue I still have with this PR.  Do you really need to allow warnings on this test?